apps/web/components/SideNav.tsx

```diff
@@ -392,14 +398,15 @@ function Links(props: { path: string; links: NavItem[] }) {
 
 function NavLink(props: { path: string; link: NavItem }) {
   const { link } = props;
+  const isActive = props.path === link.href || props.path.includes(link.href);
```

Comment:
What was the reason for `includes` here?
But also `includes` would match `===` too.

Reply:
The "includes" matched the query string, "===" doesn't match the query string it is for the paths.

Reply:
If `props.path.includes(link.href)` then it must be that `props.path === link.href`

Reply:
Ohh I haven't checked that yet! If that's the case I'll make the changes